app/src/main/res/layout/activity_sign_in.xml

```diff
@@ -2,103 +2,166 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/main_container"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/white"
+    android:fitsSystemWindows="true"
     android:orientation="vertical">
 
     <ImageView
         android:id="@+id/background"
         android:layout_width="match_parent"
-        android:layout_height="@dimen/sign_in_background_height"
+        android:layout_height="match_parent"
+        android:contentDescription="@string/background_image"
         android:scaleType="fitXY"
+        android:src="@drawable/splash_screen_background"
         android:visibility="visible"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:srcCompat="@drawable/sign_in_background"
-        android:contentDescription="@string/background_image" />
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <View
```

Comment:
Avoid duplicate code by creating a layout with the background image and the overlay color that will be reused in all screens